problem_builder/mentoring.py

```diff
@@ -156,18 +161,28 @@ def get_content_titles(self):
             return [self.display_name]
         return []
 
-    def get_theme(self):
+    def get_settings(self, settings_key, default):
```

Comment:
Perhaps it would be best to take advantage of the XBlock settings mixin from utils.

https://github.com/edx/xblock-utils/blob/3b58c757f06943072b170654d676e95b9adb37b0/xblockutils/settings.py#L11